plugins/together/newhelm/suts/together_client.py

```diff
@@ -96,7 +96,12 @@ def evaluate(
         }
         as_json = request.model_dump(exclude_none=True)
         session = requests.Session()
-        retries = Retry(total=5, backoff_factor=0.5, status_forcelist=[503, 429])
+        retries = Retry(
+            total=6,
+            backoff_factor=2,
+            status_forcelist=[503, 429],
+            allowed_methods=["POST"],
+        )
         session.mount("https://", HTTPAdapter(max_retries=retries))
```

Comment:
This can technically probably also be `session.mount(self._URL, HTTPAdapter(max_retries=retries)`, but no need to overthink this right now. Thanks for getting this one resolved.